pkg/api/handlers/libpod/images.go

```diff
@@ -487,14 +487,32 @@ func CommitContainer(w http.ResponseWriter, r *http.Request) {
 		SystemContext:         sc,
 		PreferredManifestType: mimeType,
 	}
+	if r.Body != nil {
```

Comment:
can you combine the duplicate code between the libpod/compat endpoint into function instead of copy pasting please

Reply:
Moved that into `pkg/domain/infra/abi.DecodeOverrideConfig()`.  LMK if there's a better place for it.